task/tkn-bundle/0.1/tkn-bundle.yaml

```diff
@@ -100,9 +100,9 @@ spec:
         fi
       done
 
-      if [[ -n "$(params.STEPS_IMAGE)" ]]; then
+      if [[ -n "${STEPS_IMAGE}" ]]; then
```

Comment:
Why change it from a param to an env var?

Reply:
I should've added to the commit message: https://tekton.dev/docs/pipelines/tasks/#substituting-in-script-blocks

The yaml-lint task was actually raising those errors.

Reply:
Ah, I didn't know that. thanks!